components/Countdown/Countdown.tsx

```diff
@@ -68,12 +69,13 @@ export const Countdown = ({
 
   return (
     <Grid
+      {...props}
```

Comment:
Passing the ...props into the root of the Countdown component so it can be clicked on in the visual editor